kie-server-parent/kie-server-services/kie-server-services-jbpm-ui/src/main/java/org/kie/server/services/jbpm/ui/ImageServiceBase.java

```diff
@@ -43,7 +43,7 @@
 import static org.jbpm.process.svg.processor.SVGProcessor.COMPLETED_BORDER_COLOR;
 import static org.jbpm.process.svg.processor.SVGProcessor.COMPLETED_COLOR;
 import static org.kie.server.api.KieServerConstants.KIE_SERVER_IMAGESERVICE_MAX_NODES;
-
+import static org.jbpm.services.api.RuntimeDataService.EntryType.*;
```

Comment:
Generally speaking star imports should be omitted.

Can we import (not-statically) just the EntryType class and refer to the values as `EntryType.START.getValue()` instead?

Reply:
ok

Reply:
done